data/schema/schema_v1.json

```diff
@@ -514,7 +514,8 @@
                                     "$ref": "#/definitions/q_code"
                                   },
                                   "child_answer_id": {
-                                    "type": "string"
+                                    "type": "string",
+                                    "description": "Response to follow up question resulting from an answer."
```

Comment:
This isn't very clear, and I think needs more than a short description, something like 'The id of an answer that should be treated as a child of this one, for example a text field answer can be the child of a radio option. The id must be of an answer in this list of answers.'

Reply:
Done, should we add a description for Parent_Id, perhaps an inverse of the child_answer_id,

'The id of an answer which is the parent of one or more child answers, for example a radio option leading to a text field answer.'

Reply:
I can't see `parent_id` in the schema

Reply:
sorry should have written parent_answer_id, it was getting late and schema blind